doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -314,6 +343,36 @@ messages and additional metadata helpful for debugging.
+<!-- |-------------------------|------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| -->
+<!-- | `id`                    | [`uuid`]                     | The ID of the execution event.                                                                                                                                                                                                 | -->
+<!-- | `prepared_statement_id` | [`uuid`]                     | The ID of the prepared statement being executed. Corresponds to [`mz_prepared_statement_history.id`](#mz_prepared_statement_history).                                                                                          | -->
+<!-- | `sample_rate`           | [`double precision`]         | The sampling rate at the time the execution began.                                                                                                                                                                             | -->
+<!-- | `params`                | [`text list`]                | The values of the prepared statement's parameters.                                                                                                                                                                             | -->
+<!-- | `began_at`              | [`timestamp with time zone`] | The time at which execution began.                                                                                                                                                                                             | -->
+<!-- | `finished_at`           | [`timestamp with time zone`] | The time at which execution ended.                                                                                                                                                                                             | -->
+<!-- | `was_successful`        | [`bool`]                     | Whether the statement executed successfully. Mutually exclusive with `was_canceled` and `was_aborted`.                                                                                                                         | -->
+<!-- | `was_canceled`          | [`bool`]                     | Whether the statement execution was canceled. Mutually exclusive with `was_successful` and `was_aborted`.                                                                                                                      | -->
+<!-- p| `was_aborted`           | [`bool`]                     | Whether the statement execution was aborted. "Aborted" means that the execution had not finished when Materialize restarted (for maintenance or due to an error). Mutually exclusive with `was_successful` and `was_canceled`. | -->
```

Comment:
How are we able to detect abortions if they deal with the system restarting.

Reply:
On startup, we re-read this collection from the stash (currently. We will probably switch to using persist directly rather than the stash). At that time we fill in "aborted" for any non-finished value.